### PR TITLE
feat(fgs/function): add encrypted user data support

### LINKED ISSUE #1763: [FGS] There is no way to pass sensitive information to huaweicloud_fgs_function resource
There is no way to pass sensitive information to huaweicloud_fgs_function resource. 
The only available huaweicloud_fgs_function parameter for passing user key-value pairs is user_data.
When I pass sensitive data over there, both sensitive and regular info go to Environment Variables section undo Configuration tab, even though I marked sensitive variables

### Terraform Version
Terraform v1.0.11
on darwin_amd64
+ provider registry.terraform.io/huaweicloud/huaweicloud v1.26.1

### Terraform Configuration Files
in variables.tf:
```
variable "REDIS_PASSWORD" {
  type      = string
  sensitive = true
}
```

in terraform.tfvars
```
REDIS_PASSWORD = "MY_SECURE_PASSWORD"
```

in main.tf:
```
user_data = jsonencode({
    REDIS_HOST     = var.REDIS_HOST
    REDIS_PORT     = var.REDIS_PORT
    REDIS_PASSWORD = var.REDIS_PASSWORD
  })
```

### Expected Behavior
sensitive information must go to Encryption Settings section, not in Environment Variables section (in UI)

### Actual Behavior
sensitive information goes to Environment Variables section along with regular user data

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
The function does not support encrypted user data yet and only support normal environment variables setting (user data).

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #1763

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add encrypted user data support
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go [hunk 1]
```diff
@@ -363,6 +369,7 @@ func resourceFgsFunctionV2Read(d *schema.ResourceData, meta interface{}) error {
 		d.Set("runtime", f.Runtime),
 		d.Set("timeout", f.Timeout),
 		d.Set("user_data", f.UserData),
+		d.Set("encrypted_user_data", f.UserData),
```
Comment (niuzhenguo) [inline huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go]: should be `f.EncryptedUserData`

Comment (Lance52259) [reply huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go]: OFC

**Comment by niuzhenguo** [comment]
need to rebase it. otherwise looks good to me.

**Pull request merged**